homeassistant/components/nzbget/manifest.json

```diff
@@ -3,5 +3,6 @@
   "name": "NZBGet",
   "documentation": "https://www.home-assistant.io/integrations/nzbget",
   "requirements": ["pynzbgetapi==0.2.0"],
-  "codeowners": ["@chriscla"]
+  "codeowners": ["@chriscla"],
```

Comment:
You should add yourself as a code owner with this level of refactoring

Reply:
@bdraco i wasnt really looking to maintain this one as i dont personally use it to be a good advocate/tester for it. it was a help the community project based on another PR i had to reject due to one of the ADRs (monitored conditions) and then seeing how it could bstter use DataUpdateCoordinator and config entries